src/libraries/Native/Unix/System.Security.Cryptography.Native.Apple/pal_signverify.c

```diff
@@ -235,59 +92,190 @@ static int32_t ConfigureSignVerifyTransform(SecTransformRef xform,
         return 0;
     }
 
-    if (includeHashAlgorithm)
-    {
-        CFStringRef cfHashName = NULL;
-        int32_t hashSize = 0;
+    return 1;
+}
+#endif
 
+// Legacy algorithm identifiers
+extern const SecKeyAlgorithm kSecKeyAlgorithmRSASignatureDigestPKCS1v15MD5; // = CFSTR("algid:sign:RSA:digest-PKCS1v15:MD5");
+
+static CFStringRef GetSignatureAlgorithmIdentifier(PAL_HashAlgorithm hashAlgorithm, PAL_SignatureAlgorithm signatureAlgorithm)
+{
+    if (signatureAlgorithm == PAL_SignatureAlgorithm_EC)
+    {
+        // ECDSA signatures are always based on digests. The managed implementation
+        // will always pre-hash data before getting here.
+        return kSecKeyAlgorithmECDSASignatureDigestX962;
+    }
+    if (signatureAlgorithm == PAL_SignatureAlgorithm_RSA_Pkcs1)
+    {
         switch (hashAlgorithm)
         {
-            case PAL_MD5:
-                cfHashName = kSecDigestMD5;
-                break;
-            case PAL_SHA1:
-                cfHashName = kSecDigestSHA1;
-                break;
-            case PAL_SHA256:
-                cfHashName = kSecDigestSHA2;
-                hashSize = 256;
-                break;
-            case PAL_SHA384:
-                cfHashName = kSecDigestSHA2;
-                hashSize = 384;
-                break;
-            case PAL_SHA512:
-                cfHashName = kSecDigestSHA2;
-                hashSize = 512;
-                break;
-            default:
-                return kErrorUnknownAlgorithm;
+            case PAL_MD5: return kSecKeyAlgorithmRSASignatureDigestPKCS1v15MD5;
+            case PAL_SHA1: return kSecKeyAlgorithmRSASignatureDigestPKCS1v15SHA1;
+            case PAL_SHA256: return kSecKeyAlgorithmRSASignatureDigestPKCS1v15SHA256;
+            case PAL_SHA384: return kSecKeyAlgorithmRSASignatureDigestPKCS1v15SHA384;
+            case PAL_SHA512: return kSecKeyAlgorithmRSASignatureDigestPKCS1v15SHA512;
```

Comment:
Did `clang-format -i` do this?

Reply:
It comes from earlier Kevin's code. I will try running clang-format.

Reply:
Oh yeah I am... a really bad example of properly formatting code.

Reply:
I had my era of 3 space indentation so I don't think I am any better at this...